### ISSUE TITLE
 MultilineOperationIndentation cop broken?

### ISSUE THREAD

**Issue by vinagrito** [bbatsov/rubocop#3916]
I'm on `0.46.0` version and I have configured:
```yml
Style/MultilineOperationIndentation:
  EnforcedStyle: indented
```
Yet, when having this:
```ruby
GalleryImage.where(account_id: current_account.id)
  .order(id: :desc)
  .paginate(page: params[:page], per_page: PER_PAGE)
```

It reports:
`Align .order with .where`
`Align .paginate with .where`

--------

## RuboCop version
``` 
$ bundle show rubocop
 /.bundle/gems/rubocop-0.46.0
```

Could I be missing something obvious here?

**Comment by mikegee** [comment]
Either run Rubocop with `--display-cop-names` or put this in your `.rubocop.yml`:

```yaml
AllCops:
  DisplayCopNames: true
```

That way you get much more helpful output:

```
Offenses:

test/test.rb:2:3: C: Style/MultilineMethodCallIndentation: Align .order with .where on line 1.
  .order(id: :desc)
  ^^^^^^
test/test.rb:3:3: C: Style/MultilineMethodCallIndentation: Align .paginate with .where on line 1.
  .paginate(page: params[:page], per_page: PER_PAGE)
  ^^^^^^^^^
```

All that to say: `MultilineOperationIndentation != MultilineMethodCallIndentation`

**Comment by bbatsov** [comment]
I'm closing this as a configuration issue.

**Comment by vinagrito** [comment]
thx @mikegee 🙇. Indeed config issue @bbatsov, my bad.
